File: app/src/main/java/tool/xfy9326/naucourse/Views/RecyclerViews/LevelExamViewHolder.java
package tool.xfy9326.naucourse.Views.RecyclerViews;

import android.support.annotation.NonNull;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import tool.xfy9326.naucourse.R;

class LevelExamViewHolder extends RecyclerView.ViewHolder {
    final TextView textView_level_exam_name;
    final TextView textView_level_exam_type;
    final TextView textView_level_exam_term;
    final TextView textView_level_exam_score_one;
    final TextView textView_level_exam_score_two;
    final TextView textView_level_exam_ticket;
    final TextView textView_level_exam_certificate;

    LevelExamViewHolder(@NonNull View view) {
        super(view);
        textView_level_exam_name = view.findViewById(R.id.textView_level_exam_name);
        textView_level_exam_type = view.findViewById(R.id.textView_level_exam_type);
        textView_level_exam_term = view.findViewById(R.id.textView_level_exam_term);
        textView_level_exam_score_one = view.findViewById(R.id.textView_level_exam_score_one);
        textView_level_exam_score_two = view.findViewById(R.id.textView_level_exam_score_two);
        textView_level_exam_ticket = view.findViewById(R.id.textView_level_exam_ticket);
        textView_level_exam_certificate = view.findViewById(R.id.textView_level_exam_certificate);

    }
}

File: app/src/main/java/tool/xfy9326/naucourse/ui/views/recyclerview/viewholders/LevelExamViewHolder.kt
package tool.xfy9326.naucourse.ui.views.recyclerview.viewholders

import androidx.recyclerview.widget.RecyclerView
import com.google.android.material.textview.MaterialTextView
import tool.xfy9326.naucourse.databinding.ViewLevelExamItemBinding

class LevelExamViewHolder(binding: ViewLevelExamItemBinding) : RecyclerView.ViewHolder(binding.root) {
    val tvLevelExamName: MaterialTextView = binding.tvLevelExamName
    val tvLevelExamType: MaterialTextView = binding.tvLevelExamType
    val tvLevelExamTerm: MaterialTextView = binding.tvLevelExamTerm
    val tvLevelExamGrade1: MaterialTextView = binding.tvLevelExamGrade1
    val tvLevelExamGrade2: MaterialTextView = binding.tvLevelExamGrade2
    val tvLevelExamTicketNum: MaterialTextView = binding.tvLevelExamTicketNum
    val tvLevelExamCertificateNum: MaterialTextView = binding.tvLevelExamCertificateNum
    val tvLevelExamNote: MaterialTextView = binding.tvLevelExamNote
}